new-content/events/summer-school-2021.md

```diff
@@ -258,3 +245,47 @@ faq:
   - question: 'Still have more questions?'
     answer: 'For any questions about the summer school, please submit your questions using the form below. For all other enquiries, feel free to email us directly at <a href="mailto:qiskit.events@us.ibm.com" target="_blank" style="cursor:pointer; text-decoration: none; color: #0f62fe;">qiskit.events@us.ibm.com</a>.'
 ---
+
+<event-template :about="about" :agenda="agenda">
+  <template #title>
+  Qiskit Global Summer School 2021:<br/>
+  Quantum Machine Learning
+  </template>
+  <template #description>
+
+  The Qiskit Global Summer School 2021 is a two-week intensive summer school designed to empower the next generation of quantum researchers and developers with the skills and know-how to explore quantum applications on their own. This second-annual course, made up of twenty lectures, five applied lab exercises, hands-on mentorship, and live Q&A sessions, focuses on developing hands-on experience and understanding of quantum machine learning.
+
+  Registrations will open soon, and be available on a first-come, first-serve basis with global distribution. For any questions, please check out our FAQ or submit an enquiry using the form below!
+
+  </template>
+  <template #call-to-actions>
+    <app-cta
+      label="Learn More"
+      url="http://qisk.it/QGSS2021EnquiryForm"
+      :segment="{ action: 'qgss2021-enquiry-form' }"></app-cta>
+  </template>
+  <template #faq-topics>
+  <cv-accordion-item>
+  <template #title>
+    What are the pre-requisites for enrolling in the Summer School?
+  </template>
+  <template #content>
+
+  Minimal prerequisites are required for the Qiskit Global Summer School. If you know <app-link url="https://www.khanacademy.org/math/precalculus/x9e81a4f98389efdf:matrices/x9e81a4f98389efdf:multiplying-matrices-by-matrices/v/matrix-multiplication-intro">how to multiply two matrices</app-link>, and have some programming experience in Python, you are ready for the Summer School.
```

Comment:
Not sure if using qiskit.org components would have worked with the previous method but this surely work and there is no need of using HTML `a` tags when we can use our `AppLink` components.